browser/ui/webui/brave_welcome_ui.cc

```diff
@@ -112,15 +113,24 @@ void WelcomeDOMHandler::HandleRecordP3A(const base::ListValue* args) {
   RecordP3AHistogram(screen_number_, finished_);
 }
 
+// Converts Chromium country ID to 2 digit country string
+// For more info see src/components/country_codes/country_codes.h
+std::string CountryIDToCountryString(int country_id) {
+  char chars[3] = {0};
+  chars[1] = country_id & 255;
+  chars[0] = (country_id >> 8) & 255;
+  return std::string(chars);
```

Comment:
nit: whenever I do bitwise operations, I assume that I'll get it wrong and add extra asserts ;)

In this case, for example, what I might do is something like:
```
std::string ret(chars);
assert(ret.size() == 2);
return ret;
```

Reply:
Addressed with https://github.com/brave/brave-core/pull/8966/commits/365faf4b646bc58fc3e7a2984f301503e1cc5d8c 😄 

Made it a little cleaner too